models/classes/user/UserService.php

```diff
@@ -48,4 +50,20 @@ public function getUserById($userId)
         }
         return $user;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function addUser(
+        $login,
+        $password,
+        Resource $role = null,
+        TypeClass $class = null
```

Comment:
$class isn't used

Reply:
I have to follow the interface signature.